dev/com.ibm.websphere.rest.handler/resources/com/ibm/wsspi/rest/config/resources/CWWKOMessages.nlsprops

```diff
@@ -0,0 +1,37 @@
+###############################################################################
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.webshpere.rest.handler/resources/com/ibm/wsspi/rest/config/resources/CWWKOMessages.nlsprops
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#
+#COMPONENTPREFIX CWWKO
+#COMPONENTNAMEFOR CWWKO Rest Handler
+#
+# NLS_MESSAGEFORMAT_VAR
+#
+#   Strings in this file which contain replacement variables are processed by the MessageFormat 
+#   class (single quote must be coded as 2 consecutive single quotes ''). Strings in this file 
+#   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
+#   (single quote must be coded as one single quote '). 
+
+# All messages must use the range CWWKO1500 to CWWKO1529
+
+CWWKO1500_NOT_FOUND=CWWKO1500E: Did not find any configured instances of {0}.
+CWWKO1500_NOT_FOUND.explanation=The requested configuration element could not be found in the server configuration.
+CWWKO1500_NOT_FOUND.useraction=Verify that the configuration element exists in server configuration and that it is correctly spelled with matching case.
+
+CWWKO1501_INVALID_IDENTIFIER=CWWKO1501E: Unique identifier {0} is not valid. The correct unique identifier might be: {1}.
```

Comment:
Change
"Unique identifier {0} is not valid. The correct unique identifier might be: {1}."
To
"The {0} unique identifier is not valid. The correct unique identifier might be {1}."